### DISCUSSION TITLE
How to configure for rendering width of spacing character is the same on editor

### DISCUSSION THREAD

**Discussion by nguyenhmtriet** [SixLabors/ImageSharp.Drawing#218] [Q&A] [answered]
Hi guys, It supposes to have this collection of string, I copy it from a template, so basically it will return result format as below.

![carbon](https://user-images.githubusercontent.com/24971905/163596248-103cadf0-51b1-4d02-a9b4-ba54eb1ef28f.svg)

But It ends up like this because of width spacing character I think. I also try different fonts and results are the same. Additional to this, I also use scale font size, it still not working.

**Font size as setting**
![pic](https://user-images.githubusercontent.com/24971905/163596752-b551fb77-626c-4317-8539-18c9b2640719.png)

**Scaled font size**

![pic](https://user-images.githubusercontent.com/24971905/163596910-203d7b52-3408-4d4c-9578-c470f3aa0a90.png)

The question is how I can align or set width for spacing character as the same on editor. Thanks in advance.

**Comment by tocsoft** [comment, marked as answer]
you have to first use a Monospaced font

**Reply by nguyenhmtriet** [reply]
Hi, thank you for pointing out this.
